### PR TITLE
Feature - Load enterprise APISpec

### PR DESCRIPTION
**Pull request by khaitruong922**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -3,6 +3,10 @@ skipsdist = true
 isolated_build = true
 envlist = lint, coverage
 
+[pytest]
+filterwarnings =
+    ignore::DeprecationWarning
+
```
Comment (jrobinAV) [inline tox.ini:9]: WHy do we ignore the deprecation ?

**Review by jrobinAV — APPROVED** [review]

File: tox.ini (hunk 1 above)
Comment (khaitruong922) [reply tox.ini:9]: Some of the tests still use the old API, for example: `path` instead of `default_path`

Comment (jrobinAV) [reply tox.ini:9]: 👍🏻

**Comment by jrobinAV** [comment]
This fix must also be merged into branch 1.1, right ?

**Pull request merged**